client/src/components/modal/mosaic_dynamic.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const ModalListItem = ({photos}) => {
+  const [firstURL, ...otherTwo] = photos;
```

Comment:
Reviewing:  client/src/components/modal/mosaic_dynamic.jsx

Code looks good, like the implementation of: const [firstURL, ...otherTwo] = photos; #FANCY